agent/credentials/manager.go

```diff
@@ -38,6 +38,10 @@ const (
 
 	v1CredentialsEndpointRelativeURIFormat = "%s?" + CredentialsIDQueryParameterName + "=%s"
 	v2CredentialsEndpointRelativeURIFormat = "%s/%s"
+
+	// The role types specified in the credentials payload msg from the backend
+	ApplicationRoleType = "TaskApplication"
```

Comment:
should be `"// ApplicationRoleType specifies ..."`

Reply:
fixed.